draft-ietf-ipsecme-rfc8229bis.xml

```diff
@@ -1356,6 +1356,11 @@
         </section>
 
         <section title="Acknowledgments" numbered="no" anchor="acknowledgments">
+          <t>The following people authored RFC8229: Tommy Pauly, Samy Touati and Ravi Mantha.
+          Since this document is an update for RFC 8229, most of its text was borrowed
+          from this RFC.
```

Comment:
```suggestion
          Since this document updates and obsoletes RFC 8229, most of its text was borrowed
          from the original document.
```